components/blocks/badge/_badge.scss

```diff
@@ -0,0 +1,22 @@
+.badge {
+    font-size: 12px;
+    width: auto;
+    position: absolute;
+    padding: 5px 10px;
+    top: 0;
+
+    &--new {
+        background: $color-accent;
+        color: $color-white;
+    }
+
+    &--alert {
+        background: $color-aqua;
+        color: $color-white;
+    }
+
+    &--error {
+        background: $color-main1;
```

Comment:
Oh damn, what is wrong with the variables... All of the `color-some-name-of-color` and `color-name-some-random-numer`, shouldn't exist.